### ISSUE TITLE
Games will not run

### ISSUE THREAD

**Issue by initioterum** [itchio/itch#1554]
Hi guys, I'm having an issue on my Late '13 MacBook ProR 13 whereby many of my downloaded games will simply not run. There is no error message - the icon will just pop up and then disappear again. I have tried uninstalling and reinstalling the games but it makes no difference. I have also reinstalled the itch app, but that didn't fix it either. 

It's not the case for all games though.. e.g. Roguelight works but 2064 does not (even though I have played it previously).

Do you have any suggestions for what might fix it?

Thanks!

**Comment by fasterthanlime** [comment]
Could you post your itch app log?

It should be at `~/Library/Application Support/itch/logs/itch.log.txt`

**Comment by initioterum** [comment]
Here you are, thanks for looking into it! 
[itch.txt](https://github.com/itchio/itch/files/1404394/itch.txt)

**Comment by fasterthanlime** [comment]
Thanks for replying! I don't see anything suspicious for the logs - does it do the same with any freshly-installed games or just with 2064 ? Does Royals run for example? https://ashervo.itch.io/royals

**Comment by initioterum** [comment]
I just downloaded and tried to run Royals but it didn't run either. Logs: 
[itch.txt](https://github.com/itchio/itch/files/1404411/itch.txt)

I tried a few more in my library, seems like Roguelight is the exception - can't find any others that work

**Comment by fasterthanlime** [comment]
Can you right click on the tab Royals is open in, click Advanced -> View logs and upload that as well?

Also: if you click "Show contents" and open Royals.app, does it launch properly?

**Comment by initioterum** [comment]
[Royals logs](https://github.com/itchio/itch/files/1404417/cave-7f9f82d6-d5ce-4c4a-9353-014131c5b740.txt)

Attempting to open Royals by opening the .app from package contents doesn't launch it

**Comment by fasterthanlime** [comment]
> Attempting to open Royals by opening the .app from package contents doesn't launch it

At this ponit, I'm fairly sure the problem is not with the app.

Can you try downloading Royals from the itch.io website, extracting it and running it? If that fails, I'd start looking elsewhere for the problem :)

Have you already tried rebooting? That's more of a Windows solution but the ways of macOS are deep and impenetrable sometimes too..

**Comment by initioterum** [comment]
That didn't work either unfortunately. Yeah it's been like this for a couple of months or something now, so seems like it's here to stay. Thanks for your suggestions and sorry for dragging you into this bizarre issue! I'll see what I can find elsewhere

**Comment by fasterthanlime** [comment]
No problem - don't forget to post back when you figure out the issue! It can be very useful if someone has a similar issue in the future :)

**Comment by initioterum** [comment]
Okay I finally got some time looking at this and I worked out what the issue was. The Users/[username]/Library/Logs folder had been changed and I didn't have permissions to read or write to the folder. Giving myself permission (Get Info > Unlock the padlock > + [username] (read & write)) solved the problem straight away.

For anyone having a similar but not identical I worked this out using the Console app. I tried to open Royals, did a search for 'royals' and then had a look at where things started to go wrong:

`File creation failed [/Users/...] Permission denied`
